docs/content/configuration/index.md

```diff
@@ -42,6 +42,9 @@ We recommend just setting the base ZK path and the ZK service host, but all ZK p
 |--------|-----------|-------|
 |`druid.zk.paths.base`|Base Zookeeper path.|`/druid`|
 |`druid.zk.service.host`|The ZooKeeper hosts to connect to. This is a REQUIRED property and therefore a host address must be supplied.|none|
+|`druid.zk.service.user`|The username to authenticate with ZooKeeper. This is an optional property.|none|
+|`druid.zk.service.pwd`|The [Password Provider](../operations/password-provider.html) or the string password to authenticate with ZooKeeper. This is an optional property.|none|
+|`druid.zk.service.authScheme`|The authentication scheme as supported and honored by ZooKeeper.|digest|
```

Comment:
Are there any ZK auth schemes other than "digest" that have "user:password" auth data format? If not, let's note here that only "digest" is supported.

Reply:
you are right. It'`digest` and `kerebros` are the only authentication supported. I'll update the doc as `digest` is the only supported.